### PR TITLE
Add lecture materials on aggregation in SQL

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

File: materials/sql-aggregation.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+element: notes
+title: Basic Queries
```
Comment (brymz) [inline materials/sql-aggregation.md]: Aggregation?

File: materials/sql-aggregation.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+```
+SELECT species_id, MIN(weight), MAX(weight), AVG(weight)
+FROM surveys
+GROUP BY species_id;
+```
+
+* We can group by multiple columns as well.
+
+```
+SELECT species_id, plot_id MIN(weight), MAX(weight), AVG(weight)
```
Comment (brymz) [inline materials/sql-aggregation.md]: `..., plot_id, MIN(weight), ...`

**Comment by brymz** [comment]
Looks good. Just a typo and a missed(?) `title:`. 

Curious, does SQL automatically remove missing values for aggregation functions? Something I was thinking might want to be added as a note.

**Comment by ethanwhite** [comment]
Fixed.

**Pull request merged**